community/tools/ra-data-hasura/src/index.js

```diff
@@ -32,6 +30,32 @@ export default (serverEndpoint, headers) => {
       throw new Error(JSON.stringify({'error': 'Invalid table/schema resource'}));
     }
 
+    return {schema: schema, tableName: tableName};
+
+  };
+
+  const getPrimaryKey = (resource) => {
+
```

Comment:
Nitpicking here: You can remove this extra blank line here; it doesn't conform to the rest of the code's style